etsin_finder/frontend/js/components/qvain/index.jsx

```diff
@@ -179,8 +207,12 @@ class Qvain extends Component {
   }
 
   handleUpdate = e => {
-    e.preventDefault()
-    this.setState({ submitted: true })
+    // e.preventDefault()
```

Comment:
Same question as for the handleCrate menthod

Reply:
This should be resolved with commit b4eb52a